packages/datadog-instrumentations/src/helpers/instrument.js

```diff
@@ -24,7 +24,15 @@ exports.addHook = function addHook ({ name, versions, file }, hook) {
   if (!instrumentations[name]) {
     instrumentations[name] = []
   }
-
+  // if pkgName starts with node:* prefix add the part after the prefix as an instrumentation also
+  // ie: node:dns & dns
+  if (name.startsWith('node:')) {
```

Comment:
It would be cleaner to convert `name` to an array and then concat the additional value so that the rest of the code can just run in a loop.

Reply:
Also, I feel like being explicit here instead of handling `node:` automatically would be easier to read, more predictable to the reader, and more flexible.